man/systemd.timer.xml

```diff
@@ -124,7 +124,7 @@
               </row>
               <row>
                 <entry><varname>OnBootSec=</varname></entry>
-                <entry>Defines a timer relative to when the machine was booted up.</entry>
+                <entry>Defines a timer relative to when the machine was booted up. In containers, this is mapped to <varname>OnStartupSec=</varname>, so both are equivalent.</entry>
```

Comment:
this should clarify that this is only done for the system instance of systemd, not per-user instances running inside a container.

Reply:
Updated.